### DISCUSSION TITLE
How do you want to interrupt the network? For example, look at the output of backbone, neck and head?

### DISCUSSION THREAD

**Discussion by vansin** [open-mmlab/mmengine#571] [Q&A]
请问下想要给网络打断点怎么弄呀？比如看backbone，neck，head的输出 ?
How do you want to interrupt the network? For example, look at the output of backbone, neck and head?

**Comment by KeyuLi** [comment]
用ipdb/pdb在detector下面的forward中，打断点，再看每一个module的输出
